OCCT/Visualization/__init__.py

```diff
@@ -16,4 +16,15 @@
 # You should have received a copy of the GNU Lesser General Public
 # License along with this library; if not, write to the Free Software
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301 USA
-from OCCT.Visualization.Basic import *
+
+def has_wx():
+    try:
+        import wx
+        return True
+    except ImportError:
+        return False
+
+if has_wx():
```

Comment:
should it use one or the other? or should we just have two options, "BasicViewWx" and "BasicViewerQt" in case someone cares about the backend? should wx, qtpy, and PySide2 be put as run requirements in meta.yaml? all those packages are available on conda-forge, and if they get installed, then things should just work?